File: app/src/main/java/com/example/json_parser/annotations/annotation/java/Review.java
package com.example.json_parser.annotations.annotation.java;

public @interface Review {

    ReviewStatus status() default ReviewStatus.PENDING;
    String comments() default "";

    // ReviewStatus enum is a member of the Review annotation type
    public enum ReviewStatus {
        PENDING,
        FAILED,
        PASSED,
        PASSED_WITH_CHANGES
    };
}

File: app/src/main/java/com/example/json_parser/annotations/annotation/kotlin/Review.kt
package com.example.json_parser.annotations.annotation.kotlin

annotation class Review(
    val status: ReviewStatus = ReviewStatus.PENDING,
    val comments: String = ""
) {
    public enum class ReviewStatus {
        PENDING,
        FAILED,
        PASSED,
        PASSED_WITH_CHANGES
    }
}
